### PR TITLE
[HB-6185] Update Documentation for 4.4.0

### PR DESCRIPTION
**Pull request by SCastanedaMunoz**

### CODE REVIEW

**Review by pleasesavemycat — APPROVED** [review]
lgtm

File: com.chartboost.mediation/Documentation/integration/initialization.md
Comment (cinnopuu) [inline com.chartboost.mediation/Documentation/integration/initialization.md:1]: ln +22: If you would like to have more control over when to initialize the Chartboost Mediation SDK, call the following on your `Awake` method.

File: com.chartboost.mediation/Documentation/integration/configure.md
Comment (cinnopuu) [inline com.chartboost.mediation/Documentation/integration/configure.md:1]: ln +103: The custom data property is found on the `IChartboostMediationFullscreenAd` instance, and has a maximum character limit of `1000` characters. In the event that the limit is exceeded, the `customData` property will be set to null.

File: com.chartboost.mediation/Documentation/setup/settings.md
Comment (cinnopuu) [inline com.chartboost.mediation/Documentation/setup/settings.md:1]: ln 2: Starting Chartboost Mediation Unity SDK 4.4.0, we have introduced a simple Editor Window to modify your Chartboost Mediation settings. The editor can be accessed by opening `Chartboost Mediation/Configure`.

ln 7: > Make sure the `AppId` and `AppSignature` values are obtained directly from your Chartboost Mediation platform as opposed to credentials from Chartboost or any other ad network.

ln 11: The Chartboost Mediation SDK `AppId` and `AppSignature` can be set in the Editor Window, or through the available C# API as seen below:

ln 26: Note that these fields are required when calling `ChartboostMediation.StartWithOptions`. We provide this as an easy way to store and access such values.

ln 53: > When using automatic initialization, the network kill switch is disabled by default. You are required to set the `AppId` and `AppSignature` into the `ChartboostMediationSettings` scriptable object in order for this feature to work.

ln 74: ### iOS

ln 75: The value provided in the Google App Id field will be utilized by the `ChartboostMediationPostprocessor` to modify your XCode project's `Info.plist` to ensure the `GADApplicationIdentifier` exists. If the element cannot be found, then the postprocessor will add it.

ln 85: The value provided in the AppLovin SDK Key field will be utilized by the `ChartboostMediationPreprocessor` to modify your `AndroidManifest.xml` to ensure the `applovin.sdk.key` exists. If the element cannot be found, the preprocessor will add it.

ln 95: To add SKAdNetworks in your app, add the `SKAdNetworkIdentifier` and their values in your app's `Info.plist` file. This process can be manually controlled by you, however, after Chartboost Mediation 4.0.0, we have added a postprocessor to handle this process automatically if needed. 

ln 109: Starting XCode 14, bitcode has been deprecated. For more information, visit [Apple's official documentation](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes#Deprecations). We have included a setting to modify the `BITCODE_ENABLE` flag into your XCode project.

**Pull request merged**